pkg/services/object/put/distributed.go

```diff
@@ -22,15 +24,11 @@ type preparedObjectTarget interface {
 }
 
 type distributedTarget struct {
-	traversalState traversal
-	traverser      *placement.Traverser
-
-	remotePool, localPool util.WorkerPool
+	placementIterator
```

Comment:
why embedded?

Reply:
not required, just allows to directly access its fields from `distributedTarget`